spec/index.bs

```diff
@@ -4089,7 +4090,8 @@ The {{GPUTextureUsage}} flags determine how a {{GPUTexture}} may be used after i
                         - |descriptor|.{{GPUTextureDescriptor/size}}.[=GPUExtent3D/depthOrArrayLayers=] must be &le;
                             |this|.{{GPUDevice/limits}}.{{GPUSupportedLimits/maxTextureDimension3D}}.
                         - |descriptor|.{{GPUTextureDescriptor/sampleCount}} must be 1.
-                        - |descriptor|.{{GPUTextureDescriptor/format}} must not be a [=depth-or-stencil format=].
+                        - |descriptor|.{{GPUTextureDescriptor/format}} must not be a [=depth-or-stencil format=] or
+                            [=compressed format=] without [[#packed-formats|sliced 3D support]].
```

Comment:
It's slightly unclear if it's
"a (depth-or-stencil format or compressed format) without sliced 3D support" or
"a depth-or-stencil format or (compressed format without sliced 3D support)".

I think this language would be more consistent with how we usually write it, plus it can subsume the other rule:

```suggestion
                        - |descriptor|.{{GPUTextureDescriptor/format}} must support {{GPUTextureFormat/"3d"}}
                            textures according to [[#texture-format-caps]].
```

Then, also specify at the top of the `plain-color-formats` that all of those formats support 3d,
and at the top of the `depth-formats` section that none of those formats support 3d.

Reply:
Done, thanks a lot for the guidance! Certainly better in this way.